### DISCUSSION TITLE
西交利物浦大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#58] [University]
> 来自群友

先说几个前提。西浦是西式学校，没有围墙极度自由。这里的学校也基本都是找大型宿舍承包的。

**宿舍是上床下桌吗**
看情况，宿舍不是学校建的，大部分是上床下桌

**教室和宿舍有没有空调**
有，都有，宿舍24小时开空调的话大概3天50块钱的电费的样子，大教室夏天都开空调到冷死（

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
有独立卫浴。

**有早自习、晚自习吗**
没有

**有晨跑吗**
没有

**每学期跑步打卡的要求是多少公里，可以骑车吗**
没有要求

**寒暑假放多久，每年小学期有多长**
每个学期14周，寒暑假总计……120天左右？暑假3个月寒假1个月（

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
学校连墙都没有当然允许点外卖（

**学校附近有地铁站吗**
没有，得走半个钟左右才能到。最近在建

**宿舍楼有洗衣机吗**
有，去洗衣中心也有干衣机

**校园网怎么样**
破 烂。 不是这里爆炸就是那里爆炸（
只有Mail是不会爆炸的
顺带一提宿舍的网络是自己开的，三大运营商自己选百兆网线最高DL98UL40。

**每天断电断网吗，几点开始断**
宿舍独立运营，不断电不断网

**食堂价格贵吗，会吃出异物吗**
正常吃的话20块钱起步一餐……因为是自选你当然可以5块钱能搞定一餐（

**洗澡热水供应时间**
独立电热水器，有热水就能用

**校园内可以骑电瓶车吗，电池在哪能充电**
可以,宿舍可以充电

**宿舍限电情况**
普遍人均500W，具体看宿舍地区

**通宵自习有去处吗**
商务楼（

**大一能带电脑吗**
能，而且人手一台（

**学校里面用什么卡，饭堂怎样消费**
就是正常的ID card……饭堂独立运营，支持扫码/饭堂卡/cash

**学校会给学生发银行卡吗**
不会。

**学校的超市怎么样**
稍微有点贵，平时不怎么在学校买东西（没有墙当然可以到处走）

**学校的收发快递政策怎么样**
所有宿舍区都有快递点（

**学校里面的共享单车数目与种类如何**
只有苏州自己的永安行

**现阶段学校的门禁情况如何**
进出建筑需要通行码，其他地方（包括饭堂）随便走

**宿舍晚上查寝吗，封寝吗，晚归能回去吗**
不查不封能回去

**Comment by Tianjxuannnnnn** [comment]
请问建筑类是2+2还是4+0？专业授课是纯英文吗？大一新生对英语的要求是什么？大学需要考什么英语专业方面的证吗？

**Reply by kyrie8465** [reply]
建筑类2+0 4+0看你自己选，会在大二上学期进行填写意向。如果学建筑学需要大一参加天赋测试。其他学科没有测试。对大一新生没有英语要求，但是大一大二会有EAP（学术性英语）课程，每学期会考试，包含口语、论文、笔试。不过会很麻烦喔。大学期间不参与国内46级考试，自己想考需要自己去教育机构报名。目前来说英语不好的同学还是尽量别来这个学校，有同学留了3级都是因为大一EAP过不了，但是他本人专业课都是很牛的水平。

**Comment by kyrie8465** [comment]
本人西浦已毕业，欢迎感兴趣的同学提问。更正上面通宵自习去处：只要能打开门的教室你都能去通宵，个别系会有属于自己的工作室也是可以通宵的。

**Reply by Tianjxuannnnnn** [reply]
想考国内的研究生的话报考是不是不太值得

**Comment by kyrie8465** [comment]
考国内研强烈建议换学校，去个普通双一流都好。这学校考国内研几乎没有优势。但博士又不同了。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Tianjxuannnnnn ***@***.***&gt;
发送时间: 2022年6月29日 21:53
收件人: CollegesChat/university-information ***@***.***&gt;
抄送: kyrie8465 ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [CollegesChat/university-information] 西交利物浦大学 (Discussion #58)

**Reply by Tianjxuannnnnn** [reply]
他的课程是和国内大学的课程课本啥的不一样吗

**Comment by kyrie8465** [comment]
完全不一样，教材都是国外原版全英文教材。数学的课本会发国内中文版好像是一样的。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Tianjxuannnnnn ***@***.***&gt;
发送时间: 2022年6月29日 22:18
收件人: CollegesChat/university-information ***@***.***&gt;
抄送: kyrie8465 ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [CollegesChat/university-information] 西交利物浦大学 (Discussion #58)





 
他的课程是和国内大学的课程课本啥的不一样吗

**Reply by Tianjxuannnnnn** [reply]
大学选择4+0的人多吗？女生的话建议报考什么专业啊TT

**Reply by kyrie8465** [reply]
> 大学选择4+0的人多吗？女生的话建议报考什么专业啊TT

以往4+X和2+2的比例都是55开的，个别学科比例不一样。至于4+0（我的理解是不打算读硕士）比例还是比较少的，不超过10%。因为目前就业环境来说本科还是比较难找到工作的，而且学校平台优势，基本上都会选择读硕士。我的建议还是不打算读国外研究生最好别来，竞争力太差了，实话是出了长三角很少人知道。女生的话其实没有什么区别吧，学校很包容，看你个人爱好，到大二前有2次转专业的机会，个别学科会有门槛。再一个专业的话看你奔着什么方向去，就业角度建议选择未来4-6年的热门专业（具体也不好说），像我本人选的建筑，6年前就业前期和薪资都是很好的，但是毕业了就进入黑铁时代了。个人喜好角度的话西浦的课都还是比较有意思的，注重实践，做喜欢的事肯定比不喜欢的容易。如果你还有其他问题想了解更详细的话可以考虑加下我v：Kyrie_Zz
